connectors/src/connectors/confluence/lib/confluence_client.ts

```diff
@@ -241,7 +242,12 @@ export class ConfluenceClient {
       // retry the request after a delay: https://developer.atlassian.com/cloud/confluence/rate-limiting/
       if (response.status === 429) {
         if (retryCount < MAX_RATE_LIMIT_RETRY_COUNT) {
-          await setTimeoutAsync(getRetryAfterDuration(response));
+          const delayMs = getRetryAfterDuration(response);
+          logger.warn(
+            { endpoint, retryCount, delayMs },
+            "[Confluence] Rate limit hit, retrying after delay"
```

Comment:
Can we log the full headers as well it might be a bit leaky but we can see if they publish in there the actual rate limit they enforce. From there we can log it as well (their doc is a bit vague it seems)?

Reply:
leaky indeed but fine by me 👍